Extensions/mutualchecker.js

```diff
@@ -0,0 +1,83 @@
+//* TITLE Mutual Checker **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Adds an icon next to usernames to indicate if they follow you. **//
+//* DETAILS Does what it says on the tin! Doesn't work on sideblogs. Made from fresh, distilled Profiler juice. **//
+//* DEVELOPER AprilSylph **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.mutualchecker = new Object({
+
+	running: false,
+
+	add_icons: function() {
+		if ($(".follower").length > 0) {
+			$(".follower").each(function() {
+				var m_url = $(this).find(".name").text();
+				$(this).find(".name").append("<span class=\"mutualcheck " + m_url + "\"></span>");
+			});
+		}
+		var posts = XKit.interface.get_posts("mutualchecker-done");
```

Comment:
Sorry, I didn't mean to remove *all* the newlines. the ones internal to the functions/blocks were fine. There should probably be one here for readability.